src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -486,6 +486,45 @@
 short-summary: Manage Azure Comsos DB network rules.
 """
 
+
+helps['cosmosdb private-endpoint-connection'] = """
+type: group
+short-summary: Manage Azure Comsos DB private endpoint connections.
+"""
+
+helps['cosmosdb private-endpoint-connection approve'] = """
+type: command
+short-summary: Approve the specified private endpoint connection associated with Azure Comsos DB.
+"""
+
+
+helps['cosmosdb private-endpoint-connection delete'] = """
+type: command
+short-summary: Delete the specified private endpoint connection associated with Azure Comsos DB.
+"""
+
+helps['cosmosdb private-endpoint-connection reject'] = """
+type: command
+short-summary: Reject the specified private endpoint connection associated with Azure Comsos DB.
+"""
+
+
+helps['cosmosdb private-endpoint-connection show'] = """
+type: command
+short-summary: Show details of a private endpoint connection associated with Azure Comsos DB.
+"""
+
+helps['cosmosdb private-link-resource'] = """
+type: group
+short-summary: Manage Azure Comsos DB private link resources.
+"""
+
+helps['cosmosdb private-link-resource show'] = """
+type: command
+short-summary: Show the private link resources supported for Azure Comsos DB.
+"""
```

Comment:
Please provide examples here.

Reply:
should it be `list` command since you don't need to provide the name of private link resource. User just need to provide the name of account.

Reply:
Added examples.

Reply:
I changed the command to list as well.